program_info/win_install.nsi.in

```diff
@@ -4,10 +4,13 @@
 
 Unicode true
 
-Name "PolyMC"
-InstallDir "$LOCALAPPDATA\Programs\PolyMC"
-InstallDirRegKey HKCU "Software\PolyMC" "InstallDir"
+Name "@Launcher_Name@"
+InstallDir "$LOCALAPPDATA\Programs\@Launcher_Name@"
+InstallDirRegKey HKCU "Software\@Launcher_Name@" "InstallDir"
 RequestExecutionLevel user
+OutFile "../@Launcher_Name@-Setup.exe"
+
+!define MUI_ICON "../@Launcher_Branding_ICO@"
```

Comment:
Is this file copied to `build/`?

Reply:
I think so:

https://github.com/PolyMC/PolyMC/blob/9054ee18a1d472dd201cec870c77530d218167de/program_info/CMakeLists.txt#L26

Should I be using the one there instead?

Reply:
That file is copied relative to `program_info`, so it should be at `<build>/program_info/polymc.ico`. As the .nsi is also in the same directory, shouldn't the path be `./@Launcher_Branding_ICO@`?

Reply:
Oh, that's actually a little misleading. While the `.nsi` is in the build directory, the script assumes it is run from the install directory, not the build directory, e.g.

https://github.com/PolyMC/PolyMC/blob/065e38c6aa4ebde6e256b02306758d645daf525e/.github/workflows/build.yml#L204-L209

So in reality, it would be in `../[build-dir]/@Launcher_Branding_ICO@`